doc/nix-local-build.rst

```diff
@@ -268,7 +268,7 @@ this folder (the most important two are first):
     ``cabal sdist --list-only``. Thus if you do not list all your
     source files in a Cabal file, Cabal may fail to recompile when you
     edit them.
-``config`` (same format as ``cabal.project``)
+``config`` (internal binary format)
```

Comment:
I'm so sorry for rushing to a conclusion with my earlier suggestion, but now when I look at it in the context of everything else: just `binary` makes much more sense. Could you please do that instead?

Reply:
Sure!